swagger.json

```diff
@@ -46,9 +46,15 @@
           }
         }
       },
+      "/api/v1/user": {
       "put": {
         "tags": ["user router"],
         "description": "Update profile information of user with id=userId.",
+      "post": {
+        "tags": [
+          "user router"
+        ],
+        "description": "Create new user",
```

Comment:
This POST you are trying to document is messing up the documentation for the user PUT endpoint and the signup route was documented already.